sys/log/full/src/log_fcb.c

```diff
@@ -324,20 +331,15 @@ log_fcb_start_append(struct log *log, int len, struct fcb_entry *loc)
     return (rc);
 }
 
-/**
- * Calculates the number of message body bytes that should be included after
- * the entry header in the first write.  Inclusion of body bytes is necessary
- * to satisfy the flash hardware's write alignment restrictions.
- */
 static int
-log_fcb_hdr_body_bytes(uint8_t align, uint8_t hdr_len)
+log_fcb_hdr_trailer_bytes(uint16_t align, uint16_t len)
```

Comment:
should not be renamed since it apparently is still used when trailer support is disabled